src/tokens/functional/size/size.json

```
@@ -136,7 +140,8 @@
         "$type": "dimension"
       },
       "paddingBlock": {
-        "value": "calc(({control.xlarge.size} - {control.xlarge.lineBoxHeight}) / 2)"
+        "$value": "12px",
```

```suggestion
        "$value": "14px",
```

Snapshots caught this! 🎉